doc/AzurePipelines.md

```diff
@@ -0,0 +1,78 @@
+# Azure Pipelines
+
+## Pipeline Code
+This project uses CI/CD pipelines that are implemented as yaml code.
+They are declared in the following files.
+- `azure-pipelines.yml`
+
+These pipelines are divided in parameterized stages that are defined accross several files, all located under [`build/`](build/).
+The more complex stages are also divided into several steps files, again all located under the build folder.
+
+## Variable Groups and Secrets
+These pipelines rely on a few variable groups and secrets in order to fully work. These are documented in [`variables.yml`](build/variables.yml).
+
+## Main Pipeline [azure-pipelines.yml](../build/azure-pipelines.yml)
+
+At high level, the CI/CD pipelines do the following:
+- **Build** the app in **staging**.
+  - **Deploy** the staging app (to AppCenter and/or TestFlight and GooglePlay).
+- **Build** the app in **production**.
+  - **Deploy** the production app (to TestFlight and GooglePlay).
+
+It also runs automated tests during the build steps.
+
+### Pull request runs
+Due to the length of mobile builds, pipelines are configured to behave a little differently when building in a context of **pull request (PR) build validation**. To reduce the build time, some stages and steps are disabled for PR builds. This requires a specific variable called `IsLightBuild` to be set, hence why it is appearing in the pipeline. 
+
+Also, all release stages are disabled in the context of PR build validation because, with the optimizations differences, the resulting application would not represent the real thing.
+
+### Release runs
+Pipeline runs **triggered on the main branch** don't qualify for `IsLightBuild` and build the application with the goal of releasing it.
+All stages are therefore enabled.
+
+## Stages and Steps
+### Build Stage ([stage-build.yml](../build/stage-build.yml))
+This file represents the stage that builds the applications on all platforms for any specified environment.
+It has many parameters, **the most important being the following**:
+- Application environment
+- Secured files (including provisioning profiles, certificates, key stores)
+- Variables groups (containing applications ids, certificates passwords, etc.)
```

Comment:
Did you forget a comma on line 39 after certificates ?

Reply:
True, I copied from Uno Application Template, but did not notice the error.